clients.json.example

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "name": "Brave",
+    "twilioSid": "abc123",
+    "twilioToken": "xyz987"
```

Comment:
It appears as though the Twilio call logs for one of the Brave subaccounts stopped on April 4. Waiting to hear from Richard if this is related to the new version of the app. If it is, then we'll probably need to import from the old and the new accounts. This could be implemented by having two different `Brave` clients here. Or we could name them differently if we wanted to distinguish.

Reply:
Richard confirmed that he did create a new subaccount for the new version of the Brave App. Shel confirmed that she would like both of these accounts to be entered in the DB with the client name `Brave`. So that's what I'll do.